templates/factory.d.ts.hbs

```diff
@@ -0,0 +1,21 @@
+/*!
```

Comment:
nit: we probably can get rid of the banners from the templates as we already have logic to handle it in one place (https://github.com/okta/okta-sdk-nodejs/blob/master/package.json#L14)